### DISCUSSION TITLE
Add Maximize Button for compose.yaml Editor and Terminal

### DISCUSSION THREAD

**Discussion by ryzen3100** [louislam/dockge#597] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Long strings in YAML files can be difficult to read and edit due to line wrapping. This affects both the compose.yaml editor and the terminal view.

### ✔️ Solution

**Add a maximize button to:**
1. The compose.yaml editor
2. The terminal view

This would allow users to expand these elements to full-screen, making it easier to work with long lines of text and complex configurations.

**Benefits:**
- Improved readability for long YAML strings
- Easier editing of complex Docker Compose configurations
- Enhanced readability of the terminal with lengthy output

**Comment by pyxl-dev** [comment]
Hello ! I've worked on an expand panel for your feature request.

Here is the PR : https://github.com/louislam/dockge/pull/678
